dataprofiler/labelers/classification_report_utils.py

```diff
@@ -319,14 +319,14 @@ def classification_report(
     if target_names is not None and len(labels) != len(target_names):
         if labels_given:
             warnings.warn(
-                "labels size, {0}, does not match size of target_names, {1}".format(
+                "labels size, {}, does not match size of target_names, {}".format(
                     len(labels), len(target_names)
                 )
             )
         else:
             raise ValueError(
-                "Number of classes, {0}, does not match size of "
-                "target_names, {1}. Try specifying the labels "
+                "Number of classes, {}, does not match size of "
```

Comment:
Is there a reason for using "format" as opposed to f strings like in above strings?

Reply:
pyupgrade tries to clean up what it can - it's not 100% perfect. In this case it would make sense to turn the whole string into an f-string but it is figuring out that using `{0}` and `{1}` is redundant.

Happy to make a manual edit here to turn this into an f-string if you prefer.

Reply:
I'm okay either way for now, but I am more a fan of being consistent.

Reply:
No problem - I've fixed these. Others can be fixed opportunistically. FYI, pylint is configured to flag strings that can be f-strings - there's actually a lot of pylint warnings/suggestions that are flagged by the current config, they all just show up by default in VS Code.